docs/manual/guide/services/ServiceDescriptors.md

```diff
@@ -46,6 +46,10 @@ Query string parameters can also be extracted from the path, using a `&` separat
 
 @[call-query-string-parameters](code/docs/services/FirstDescriptor.java)
 
+
+When you use `call`, `namedCall` or `pathCall`, if Lagom maps that down to REST, Lagom will make a best effort attempt to map it down to REST in a semantic fashion, so that means if there is a request message, it will use `POST`, if there's none, it will use `GET`.
```

Comment:
Yes, since currently streaming requests are implemented using WebSockets, and hence GET is mandatory.  Note that with streaming requests, the message is not sent in the request body, it's sent in the WebSocket stream.

Best effort is the key here, WebSocket streaming doesn't really map down onto REST.